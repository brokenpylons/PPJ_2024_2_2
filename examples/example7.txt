for (var i = 0, 5) {
    print(i);
}

