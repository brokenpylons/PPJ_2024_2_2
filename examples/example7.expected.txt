real("8") divides("/") real("2") divides("/") real("2") 
